go/vt/vtgate/planbuilder/builder.go

```diff
@@ -60,7 +60,7 @@ func Build(query string, vschema ContextVSchema) (*engine.Plan, error) {
 	if err != nil {
 		return nil, err
 	}
-	result, err := sqlparser.RewriteAST(stmt)
+	result, err := sqlparser.PrepareAST(stmt, map[string]*querypb.BindVariable{}, "", false)
```

Comment:
we can keep the call to RewriteAST as parameterize is kept false on PrepareAST call

Reply:
changed in the latest commit.